### ISSUE TITLE
Only load files in root-node-id

### ISSUE THREAD

**Issue by Skyfay** [dweidenfeld/plexdrive#419]
I use "--root-node-id=" to determine which content is only displayed.

Well, if I use Plexdrive, all files in my Google Drive are loaded into the Cache.bolt file.

I find that a bit unnecessary, especially because I have so many files next to my videos.

Is there a way that only files that I want to play are loaded into the cache.bolt file?

**Comment by 5536086** [comment]
Yes, I had the same problem as you, --root-node-id didn't do anything, it still loads the root directory

**Comment by Skyfay** [comment]
> Yes, I had the same problem as you, --root-node-id didn't do anything, it still loads the root directory

Sorry I am very late.  If you don't want to have your whole root directory indexed, there is a way. If they are in an organization, they can create Shared Drives (Team Drives). You can then mount them using the following mount command:
`--drive-id string  |           The ID of the shared drive to mount (including team drives)`
When you do that, the entire shared drive folder is scanned, but only this one. 
But that only works if they are in an organization.
